packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -39,6 +39,21 @@ type TApplicationContextActionRights = {
   [key: string]: TActionRight;
 };
 
+type TApplicationContextGroupedByPermission = {
+  [key: string]: {
+    values: string[];
+  };
+};
+type TApplicationContextGroupedByResourceType = {
+  [key: string]: TApplicationContextGroupedByPermission;
```

Comment:
Shouldn't this be

```suggestion
  [key: string]: { values: string[] };
```

?

Reply:
no. the new is 

```js
{
  // store
  [dataFenceType]: {
    // orders
    [resourceType]: {
      // canManageOrders
      [permission]: {
        values: []
      }
    }
  }
}
```

Reply:
Exactly...have you tried my suggestion?

Reply:
Here you go: https://github.com/commercetools/merchant-center-application-kit/pull/918/commits/e7f118b8201253068c4a1cc0444fd1ad70a00339